lib/model.js

```diff
@@ -4880,6 +4881,29 @@ Model.compile = function compile(name, schema, collectionName, connection, base)
   return model;
 };
 
+Model.clientEncryption = function clientEncryption() {
+  /** @type(import('mongodb').MongoClient) */
```

Comment:
Does this comment do anything? I don't think Mongoose supports this type definition syntax